src/AdsCards.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import CardActions from '@mui/material/CardActions';
```

Comment:
Group your imports, add an empty line between React and imports from mui library